### PR TITLE
chore: create script to build KCC CLI

### PR DESCRIPTION
**Pull request by justinsb**
- **chore: generate license metadata**
- **chore: create script to build KCC CLI**

### CODE REVIEW

**Comment by acpana** [comment]
/hold 

just in case you want feedback from someone else too, feel free to cancel otherwise

/lgtm
/approve

File: dev/tasks/build-cli-bundle [hunk 1]
```diff
@@ -75,8 +75,10 @@ for GODIST in $(go tool dist list | grep "linux\|darwin\|windows" | grep "amd64\
   CGO_ENABLED=0 go build -ldflags "${LDFLAGS}" -o ${OUTPUT_PATH} github.com/GoogleCloudPlatform/k8s-config-connector/cmd/config-connector
 
   cp "${BUILD}/lib.zip" "${OUTPUT_DIR}/lib.zip"
-  cp "${BUILD}/LICENSES.txt" "${OUTPUT_DIR}/LICENSES.txt"
+  cp "${BUILD}/LICENSES.txt" "${OUTPUT_DIR}/THIRD_PARTY_NOTICES"
```
Comment (yuwenma) [inline dev/tasks/build-cli-bundle:78]: Or `THIRD-PARTY-NOTICES.txt`? (dash vs underscore)

**Review by yuwenma — COMMENTED** [review]
/lgtm

one minor

**Pull request merged**